hinosxz/chowchow
# 💥 Initialize React App

## What does this PR do?
Adds a basic front-end set up with:
- A simple architecture;
- Linting.

We still need to set up docker and the CI.

## Motivation (link to Trello card)
https://trello.com/c/MerX2WXf

## Testing Guidelines
Pull the branch, go to the `javascript` folder and run `yarn start`, you should then be able to see the React App running on port 3000.

## Release Checklist

-   [x] This code has automated tests
-   [ ] This code has documentation

## Additional Notes
